ecology.yaml

```diff
@@ -0,0 +1,546 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: 'Statistics'
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: 'Annotation'
+
+  - name: add_input_name_as_column
+    owner: mvdbeek
+    tool_panel_section_label: 'Text Manipulation'
+ 
+  - name: regex_replace
+    owner: kellrott
+    tool_panel_section_label: 'Text Manipulation'
+
+  - name: find_in_reference
+    owner: jjohnson
+    tool_panel_section_label: 'Filter and Sort'
+
+  - name: normalize_dataset
+    owner: bebatut
+    tool_panel_section_label: 'Statistics'
+    
+  - name: frogs_2_0_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+    
+  - name: frogs_3_1_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+
+  - name: xls2tabular
```

Comment:
Moving to IUC?

Reply:
This is a home made tool... Do you think a IUC guy is ok to iucize it?

Reply:
Why not, its seems general and useful enough? Maybe even as standard Galaxy converter? Disclaimer I haven't seen the tool :)

Reply:
For our ecology oriented communities this is very usefull. Extraction tabular files from names of Excel sheets. It will be really groovy if IUC can manage to iucize it!